index.html

```diff
@@ -12,123 +12,125 @@
 <body>
     <main id="main-doc">
         <div class="box">
-        <nav id="navbar">
-            <header class="h1">JS Challenges</header>
-            <a href="#Apple_and_Orange" class="nav-link">Apple and Orange</a>
-            <a href="#Breaking_best_and_worst_records" class="nav-link">Breaking best and worst records</a>
-            <a href="#Cats_and_a_mouse" class="nav-link">Cats and a mouse</a>
-            <a href="#The_hurdle_race" class="nav-link">The hurdle race</a>
-            <a href="#Simple_Array_Sum" class="nav-link">Simple Array Sum</a>
-        </nav>
-    </div>
+            <nav id="navbar">
+                <header class="h1">JS Challenges</header>
+                <a href="#Apple_and_Orange" class="nav-link">Apple and Orange</a>
+                <a href="#Breaking_best_and_worst_records" class="nav-link">Breaking best and worst records</a>
+                <a href="#Cats_and_a_mouse" class="nav-link">Cats and a mouse</a>
+                <a href="#The_hurdle_race" class="nav-link">The hurdle race</a>
+                <a href="#Simple_Array_Sum" class="nav-link">Simple Array Sum</a>
+            </nav>
+        </div>
```

Comment:
Please don't use the tab key for indentation. Use 2 white spaces as suggested in [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
You can set the number of indentation spaces on your editor settings, or in your formatted configuration file.